docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -105,7 +114,8 @@ def get_data_loaders(batch_size: int):
         dataset_train_ssl,
         batch_size=batch_size,
         shuffle=True,
-        collate_fn=collate_fn,
+        collate_fn=collate_fn if not multi_crops else swav_collate_fn,
```

Comment:
Is this a specific SwavCollateFun or a more general MultiCropCollateFun? If it is the latter, I would call it like that.